public/components/event_analytics/hooks/use_fetch_events.ts

```diff
@@ -52,7 +52,7 @@ export const useFetchEvents = ({ pplService, requestParams }: IFetchEventsParams
   ) => {
     setIsEventsLoading(true);
     return pplService
-      .fetch({ query, format }, errorHandler)
+      .fetch({ query, format }, undefined, errorHandler)
```

Comment:
?

Reply:
it's regression from MDS PR, see description and 7b5cc9f commit message:
> - update fetch call to pass in data source id 
>   - data source id is always undefined because log explorer doesn't support multi-datasource. passing undefined fixes the parameters so UI shows PPL errors in toasts